modules/apps/collaboration/message-boards/message-boards-web/src/main/resources/META-INF/resources/message_boards/edit_category.jsp

```diff
@@ -245,4 +245,95 @@ if (portletTitleBasedNavigation) {
 
 	Liferay.Util.toggleBoxes('<portlet:namespace />mailingListActive', '<portlet:namespace />mailingListSettings');
 	Liferay.Util.toggleBoxes('<portlet:namespace />outCustom', '<portlet:namespace />outCustomSettings');
+
+	AUI().use(
+		'event',
+		'node',
+		function(A) {
+			var activeCheckboxNode = A.one('#<portlet:namespace />mailingListActive');
+			var outgoingCheckboxNode = A.one('#<portlet:namespace />outCustom');
+
+			var activeCheckboxNodeChecked = activeCheckboxNode._node.checked;
+			var outgoingCheckboxNodeChecked = outgoingCheckboxNode._node.checked;
+
+			var incomingFields = [
+				'emailAddress',
+				'inServerName',
+				'inServerPort',
+				'inUseSSL',
+				'inUserName',
+				'inPassword',
+				'inReadInterval',
+				'outEmailAddress',
+				'outCustom'
+			];
+
+			var outgoingFields = [
+				'outServerName',
+				'outServerPort',
+				'outUseSSL',
+				'outUserName',
+				'outPassword'
+			];
+
+			activeCheckboxNode.on(
+				'click',
+				function(event) {
+					activeCheckboxNodeChecked = activeCheckboxNode._node.checked;
+					outgoingCheckboxNodeChecked = outgoingCheckboxNode._node.checked;
+
+					toggleDisableInputFields(incomingFields, activeCheckboxNodeChecked);
+
+					if (!activeCheckboxNodeChecked && outgoingCheckboxNodeChecked) {
+						outgoingCheckboxNodeChecked = false;
+					}
+
+					toggleDisableInputFields(outgoingFields, outgoingCheckboxNodeChecked);
+				}
+			);
+
+			outgoingCheckboxNode.on(
+				'click',
+				function(event) {
+					outgoingCheckboxNodeChecked = outgoingCheckboxNode._node.checked;
+
+					toggleDisableInputFields(outgoingFields, outgoingCheckboxNodeChecked);
+				}
+			);
+
+			function toggleDisableInputFields(fields, checkboxId) {
+				var nodeActive = false;
+
+				if (checkboxId) {
+					nodeActive = true;
+				}
+
+				for (var field in fields) {
+					var inputId = '#<portlet:namespace />' + fields[field];
+
+					var inputField = A.one(inputId);
+
+          inputField.set('disabled', false);
+
+					inputField.replaceClass('disabled', '');
+
+					if (!nodeActive) {
+						inputField.set('disabled', true);
+
+						var curClass = inputField.attr('class');
+
+						inputField.attr('class', curClass + ' disabled');
+					}
+				}
+			}
+
+			if (activeCheckboxNodeChecked) {
+				toggleDisableInputFields(incomingFields, activeCheckboxNodeChecked);
+
+				if (outgoingCheckboxNodeChecked) {
+					toggleDisableInputFields(outgoingFields, outgoingCheckboxNodeChecked);
+				}
+			}
+		}
+	);
```

Comment:
We should be able to use jQuery in this block using ```<aui:script sandbox="<%= true %>" />``` because we are not using anything specific to AUI in here.